src/DynamoCore/Search/SearchElements/NodeSearchElement.cs

```diff
@@ -148,7 +148,7 @@ public string Description
             }
         }
 
-        public string IconName
+        public virtual string IconName
```

Comment:
`string iconName` is `protected` data member, can we set `iconName = ID.ToString()` in derived class instead of making this `virtual`?